From leitao@linux.vnet.ibm.com  Thu Oct  8 15:45:00 2009
From: Breno Leitão <leitao@linux.vnet.ibm.com>
Date: Tue, 29 Sep 2009 15:16:58 -0300
Subject: jsm: Removing unused jsm_channel->ch_wopen field
To: linux-kernel@vger.kernel.org
Cc: Scott.Kilau@digi.com, greg@kroah.com
Message-ID: <552b20c8a33e0c85a9f1b188d23ce7afd2d3b21e.1254247209.git.leitao@linux.vnet.ibm.com>


Currently the jsm_channel->ch_wopen field is defined and never
used. So, this patch removes it.

Signed-off-by: Breno Leitão <leitao@linux.vnet.ibm.com>
Cc: Scott Kilau <Scott.Kilau@digi.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/jsm/jsm.h |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/serial/jsm/jsm.h
+++ b/drivers/serial/jsm/jsm.h
@@ -213,8 +213,6 @@ struct jsm_channel {
 	u8		ch_stopc;	/* Stop character		*/
 	u8		ch_startc;	/* Start character		*/
 
-	u32		ch_wopen;	/* Waiting for open process cnt */
-
 	u8		ch_mostat;	/* FEP output modem status	*/
 	u8		ch_mistat;	/* FEP input modem status	*/
 
